docs/library/README.md

```
@@ -0,0 +1,398 @@
+    fmt.Printf("Feed: %s (%d items)\n", feed.Title, len(feed.Items))
+    for _, item := range feed.Items {
+        fmt.Printf("- %s\n", item.Title)
+    }
+}
+```
+
+## 🏗️ Architecture
+
+The library provides a clean, interface-based API that wraps the core business logic without HTTP dependencies.
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)